.circleci/config.yml

```diff
@@ -179,18 +179,49 @@ jobs:
           command: |
             # Workaround for Sign Tool "private key filter" bug in Circle's Windows image.
             # Ref: https://travis-ci.community/t/codesigning-on-windows/
-            # 
-            # Fix: Import .p12 into the local certificate store. Sign Tool will use 
+            #
+            # Fix: Import .p12 into the local certificate store. Sign Tool will use
             # package.json's `certificateSubjectName` to find the imported cert.
             Import-PfxCertificate -FilePath C:\Users\circleci\simplenote-electron\resources\certificates\win.p12 -CertStoreLocation Cert:\LocalMachine\Root -Password (ConvertTo-SecureString -String $env:WIN_CSC_KEY_PASSWORD -AsPlainText -Force)
-            make package-win32 IS_WINDOWS=true
             npx electron-builder --win -p onTag
       - persist_to_workspace:
           root: C:\Users\circleci\simplenote-electron
           paths:
             - release\*.exe
             - release\*.appx
             - release\*.yml
+  windowsstore:
+    executor:
+      name: win/default
+    working_directory: C:\Users\circleci\simplenote-electron
+    environment:
+      CSC_LINK: ''
+      WIN_CSC_LINK: ''
+    steps:
```

Comment:
Instead of the somewhat unintuitive filtering in the "workflow" section, it might work to use Circle config's "when" syntax here:

```
windowsstore:
  when:
    condition: << pipeline.git.tag >>
      steps:
        // ...
```